lib/pages/user/login.dart

```diff
@@ -131,7 +131,7 @@ class _LoginPageState extends State<LoginPage> {
                                         _loginUser(context);
                                       }
                                     }
-                                  : () => null,
+                                  : null,
```

Comment:
Having a null `onPressed` shows the button as disabled which I think was the intention.